packages/grant-explorer/src/hooks/matchingEstimate.ts

```diff
@@ -0,0 +1,63 @@
+import useSWR from "swr";
+import { Address } from "viem";
+import { ChainId } from "common";
+
+export type MatchingEstimateResult = {
+  totalReceived: string;
+  contributionsCount: string;
+  sumOfSqrt: string;
+  capOverflow: string;
+  matchedWithoutCap: string;
+  matched: string;
+  difference: bigint;
+  differenceInUSD: number;
+  roundId: string;
+  chainId: number;
+  recipient: string;
+};
+
+type UseMatchingEstimatesParams = {
+  roundId: Address;
+  chainid: ChainId;
+  potentialVotes: {
+    contributor: string;
+    recipient: string;
+    amount: bigint;
+    token: string;
+  }[];
+};
+
+type JSONValue = string | number | boolean | bigint | JSONObject | JSONValue[];
+
+interface JSONObject {
+  [x: string]: JSONValue;
+}
+
+function getMatchingEstimates(
+  params: UseMatchingEstimatesParams
+): Promise<MatchingEstimateResult[]> {
+  const replacer = (_key: string, value: JSONValue) =>
+    typeof value === "bigint" ? value.toString() : value;
+
+  return fetch(
+    `${process.env.REACT_APP_ALLO_API_URL}/api/v1/chains/${params.chainid}/rounds/${params.roundId}/estimate`,
+    {
+      headers: {
+        accept: "*/*",
```

Comment:
`accept: "*/*"` here isn't really true (we definitely want JSON), was it necessary as a workaround for something?

Reply:
true, fixed.